src/content/docs/security/security-privacy/compliance/data-encryption.mdx

```diff
@@ -92,69 +92,7 @@ New Relic's disk-based encryption provides additional security while your data i
       </td>
 
       <td>
-        Free for all New Relic customers where data is stored in Amazon AWS.
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        What data is encrypted
-      </td>
-
-      <td>
-        This encryption protects the physical disk where New Relic retains your data, including the following:
-
-        * [Log management](/docs/logs/new-relic-logs/get-started/new-relic-logs-security-privacy) data
-        * [Dimensional metrics](/docs/using-new-relic/data/understand-data/new-relic-data-types#metric-time-series) data
-
-        As we implement encryption at rest for additional telemetry types, your data will be encrypted automatically with no additional steps required by you.
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        How it works
-      </td>
-
-      <td>
-        New Relic uses Amazon AWS non-volatile, memory express SSD instance store volumes for disk-level data encryption at rest. The data on each instance storage device is encrypted using an XTS-AES-256 block cipher implemented in a hardware module on the instance.
-
-        Encryption keys are generated using the hardware module and are unique to each instance storage device. All encryption keys are destroyed when the instance stops or terminates, and they cannot be recovered.
-
-        As additional security measures:
-
-        * Disk-level encryption cannot be disabled.
-        * External encryption keys cannot be provided.
-      </td>
-    </tr>
-  </tbody>
-</table>
-
-## Account-level encryption [#account-encryption]
-
-New Relic's account-level encryption at rest allows approved New Relic customers to benefit from even higher levels of security (FIPS 140-2 and FedRAMP Moderate compliant).
-
-<table>
-  <thead>
-    <tr>
-      <th style={{ width: "200px" }}>
-        Account-level encryption at rest
-      </th>
-
-      <th>
-        Comments
-      </th>
-    </tr>
-  </thead>
-
-  <tbody>
-    <tr>
-      <td>
-        Who gets it
-      </td>
-
-      <td>
-        Account-level data encryption depends on your New Relic subscription and your [account hierarchy](/docs/accounts/install-new-relic/account-setup/manage-apps-or-users-sub-accounts). For example, if your data is encrypted at the master account level, your sub-account data also is automatically encrypted at rest.
+        Encryption at rest depends on your New Relic subscription and your [account hierarchy](/docs/accounts/install-new-relic/account-setup/manage-apps-or-users-sub-accounts). For example, if your data is encrypted at the master account level, your sub-account data also is automatically encrypted at rest.
```

Comment:
"Encryption at rest depends on your New Relic subscription, New Relic's approval, and your [account hierarchy](/docs/accounts/install-new-relic/account-setup/manage-apps-or-users-sub-accounts)."

Reply:
:+1: